test/spec_helper.rb

```diff
@@ -6,10 +6,10 @@
   c.color = true
 end
 
-def fixture_path
-  File.expand_path('../fixtures', __FILE__)
-end
-
-def fixture(f)
-  File.new(File.join(fixture_path, f))
-end
+# def fixture_path
+#   File.expand_path('../fixtures', __FILE__)
+# end
+#
+# def fixture(f)
+#   File.new(File.join(fixture_path, f))
+# end
```

Comment:
any reason to not nuke this?

Reply:
nope, just commented out originally I will fix that up before merging.